### ISSUE TITLE
Null vs undefined

### ISSUE THREAD

**Issue by VladShcherbin** [tsuyoshiwada/dot-wild#10]
Hey

When we use `get` method and don't have a value by specified path, the returned value is `null`. I'd like to have this value as `undefined`, but it seems like it's not possible to do it. I tried default value as `undefined`, but since it's `undefined` it can't be set.

```js
import { get } from 'dot-wild'

let obj = {
  a: {
    c: 1
  }
}

console.log(get(obj, 'a.c', undefined)) // 1
console.log(get(obj, 'a.c.f', undefined)) // null <- here
```

Am I missing something?

**Comment by VladShcherbin** [comment]
Here is another example:

```js
import { get } from 'dot-wild'

let obj = {
  a: {
    d: undefined,
    e: null
  }
}

console.log(get(obj, 'a.d')) // undefined
console.log(get(obj, 'a.e')) // null <-- it is
console.log(get(obj, 'a.k')) // null <-- it's not
```

In this example, both paths will return `null`, but in reality, only one of them equals to `null`. So, if you'll have a check, if the returned value is `null`, only one of them will give you the correct result.

This issue and [this one](https://github.com/tsuyoshiwada/dot-wild/issues/11) bring the question, why `null` is the default returned value instead of `undefined`?

**Comment by tsuyoshiwada** [comment]
Hi @VladShcherbin Thank you for Issues! And I'm sorry that the reply was delayed.

As you say, it is not natural for `null` to return if the key is undefined ... 

https://github.com/tsuyoshiwada/dot-wild/blob/2798f212ffb3dd21043f8cd95dd61682230a2c6d/src/index.ts#L203

Can see in the above, it is due to defining the default value as `null`. I will release the modified version soon.

**Comment by tsuyoshiwada** [comment]
@VladShcherbin Released fix version as `v3.0.0` :)
Please check when you have time :pray:

**Comment by VladShcherbin** [comment]
@tsuyoshiwada this one is working now too. Thank you 👍
